--- tig.spec.orig	2015-10-18 10:34:16.695935433 +0900
+++ tig.spec	2015-10-18 10:34:37.599936153 +0900
@@ -4,7 +4,7 @@
 Summary: 	Tig: text-mode interface for git
 Name: 		tig
 Version: 	2.1.1
-Release: 	0%{?dist}
+Release: 	1%{?dist}
 License: 	GPL
 Group: 		Development/Tools
 Vendor: 	Jonas Fonseca <jonas.fonseca@gmail.com>
@@ -56,7 +56,7 @@ CFLAGS="$RPM_OPT_FLAGS -DVERSION=tig-%{v
 %{?_without_docs:  %doc doc/*.txt}
 
 %changelog
-* Sun 23 Feb 2014 Jonas Fonseca <jonas.fonseca@gmail.com>
+* Sun Feb 23 2014 Jonas Fonseca <jonas.fonseca@gmail.com>
 - Add tigrc installed in /etc
 
 * Tue Jan  8 2013 Joakim Sernbrant <serbaut@gmail.com>
